src/javascripts/ng-admin/Crud/list/maDatagridInfinitePagination.js

```diff
@@ -17,19 +17,29 @@ export default function maDatagridInfinitePagination($window, $document) {
             var perPage = parseInt(scope.perPage, 10) || 1,
                 totalItems = parseInt(scope.totalItems, 10),
                 nbPages = Math.ceil(totalItems / perPage) || 1,
-                page = 1;
+                page = 1,
+                processing = false;
             function handler() {
-                if (body.offsetHeight - $window.innerHeight - $window.scrollY < offset) {
-                    if (page >= nbPages) {
-                        return;
+                if(processing) return;
+                processing = true;
+                var timer = setInterval(function(){
```

Comment:
why do you need a `setInterval()` while the handler is already called on each scroll?